slides/background_python_and_jupyter_notebook.html

```diff
@@ -320,7 +320,7 @@
 
 ## Python Basics
 
-Please go through the [`Python basics jupyter notebook`](../../notebooks/notebooks/introduction_python.ipynb) now
+Please go through the [`jupyter notebook introduction_python.ipynb`](../../notebooks/notebooks/introduction_python.ipynb) now
```

Comment:
The user won't need to know that the file is called 'introduction_python.ipynb' as they can just use the links here and on the program page